registry/src/main.rs

```diff
@@ -6,6 +6,13 @@ use crossterm::event;
 #[cfg(debug_assertions)]
 use std::env;
 
+// #[macro_use]
+// extern crate rust_i18n;
+
+// Init translations using the `[package.metadata.i18n]` section in `Cargo.toml`
+use rust_i18n::t;
+rust_i18n::i18n!("locales");
```

Comment:
Should specify `fallback = "en-us"`

Reply:
I think that gets pulled from he package metadata, but can explicitly set it here.